### ISSUE TITLE
Build for lake (moto g7 plus)

### ISSUE THREAD

**Issue by nicobra** [lineageos4microg/docker-lineage-cicd#83]
Hey,
is it possible to provide a build for Lake?
https://download.lineageos.org/lake it's officially supported by Lineage now, but unfortunately I couldn't find a build with Microg.

**Comment by ttimasdf** [comment]
just try it yourself. this is a docker image, a building environment. Technically you could build any ROM based on AOSP. I maintained my own fork and compiled AICP、RR and even LOS for Nintendo Switch. Compiling LineageOS is the easiest case, tweaking environment variables and `docker run` is enough.

**Comment by nicobra** [comment]
Hey, I already built it myself, even started to set up my own terraform script to start an ota server instance on hetzner where I monthly get my updates from. The build script is great and everything is working fine. Thanks to the adjustments in the 17.1 PR branch.

However, this is not my point. Lake is officially supported by lineage now and nightly builds would be great, also for users that don't have the effort to build it themselves. But I also saw that this project is unfortunately not really maintained right now. My time and knowledge to set up and, more importantly, harden official build servers is unfortunately quite limited. But if there is any possibility to support this it would be great to know. Would be sad if this project dies
